src/App.css

```diff
@@ -0,0 +1,16 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@200;300;400;500;600;700;800;900&display=swap");
+
```

Comment:
- [ ] **Design:** 
- Hello, since you used React and Redux to build a **mobile web application** and not a desktop web application, I would advise you to follow the[ design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) which include: **font face, Size, and weight.**
- Please, you need to follow the layout(the space between elements)

- [ ] Please, carefully look at your project layout below. and compare to the original:
-  
![Capture1](https://user-images.githubusercontent.com/63972670/225972051-f736ee1a-760b-45a1-a27c-658db7fc76d0.PNG)
- **With different sizes**
![Capture](https://user-images.githubusercontent.com/63972670/225972272-a5197bb3-1da3-4484-a16a-66993ea93632.PNG)
- **Original design**

![1](https://user-images.githubusercontent.com/63972670/225972456-ec44ff0d-0b1b-46f0-864b-3ced63e55e89.PNG)
**NB**:  I have requested the change, based on the project requirements

Reply:
Add the lato font family

Reply:
![image](https://user-images.githubusercontent.com/105445428/225978296-c6a11374-b0e0-4604-b97d-f6076a20e8c4.png)
change the justify-content  from 'space-between' to 'center'